apply/screening-example.js

```diff
@@ -0,0 +1,118 @@
+const communicateActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-CommunicateUnstructuredRecommendations.json'));
+const cotestActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-CotestingScreeningRecommendation.json'));
+const dashboardActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-DisplayCervicalCancerMedicalHistory.json'));
+const hpvActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-PrimaryHpvScreeningRecommendation.json'));
+const surveillanceActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-SurveillanceActivityDefinition.json'));
+const treatmentActivity = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/ActivityDefinition-TreatmentActivityDefinition.json'));
+
+// Questionnaire resources
+const provideMoreInformationQuestionnaire = JSON.parse(readFileSync('fsh-tank/fsh-generated/resources/Questionnaire-ProvideMoreInformation.json'));
+
+// Bring in an example patient bundle from the test folder
+const examplePatientBundle = JSON.parse(readFileSync('test/ScreeningAverageRisk/test_results/ScreeningAverageRiskLibrary_v1.0.0/bundles/Under_30_and_have_had_recent_cytology_test.json'));
+const examplePatientResources = examplePatientBundle.entry.map(r => r.resource); // convert from bundle to flat array
+
+// Read in ELM JSON representation of CDS logic
+const elmJsonDependencyArray = [
+  JSON.parse(readFileSync('cql/TopLevelScreeningLibrary.json')),
+  JSON.parse(readFileSync('cql/ScreeningSymptomaticLibrary.json')),
+  JSON.parse(readFileSync('cql/ScreeningDesExposureLibrary.json')),
+  JSON.parse(readFileSync('cql/ScreeningImmunocompromisedLibrary.json')),
+  JSON.parse(readFileSync('cql/ScreeningAverageRiskLibrary.json')),
+  JSON.parse(readFileSync('cql/DisplayCervicalCancerMedicalHistory.json')),
+  JSON.parse(readFileSync('cql/CCSMCommonFunctions.json')),
+  JSON.parse(readFileSync('cql/CDSConnectCommonsforFHIRv401.json')),
+];
+
+// Reformat ELM JSON value set references to match what is expected by the 
+// code service built into the cql execution engine
+const elmJsonDependencies = elmJsonDependencyArray.reduce((acc, elm) => {
+  let refs = elm?.library?.valueSets?.def;
+  if (refs) {
+    refs = refs.map(r => {
+      return {
+        ...r,
+        id: r.id.split('/').pop()
+      }
+    });
+    elm.library.valueSets.def = refs;
+  }
+  return {
+    ...acc,
+    [elm.library.identifier.id]: elm
+  }
+}, {});
+
+// Read in cached value set JSON
+let valueSetJson = JSON.parse(readFileSync('test/ScreeningAverageRisk/.vscache/valueset-db.json'));
+
+// Reformat value sets to match format expected by code service built into the 
+// cql execution engine.
+valueSetJson = Object.keys(valueSetJson).reduce((acc,vs) => {
```

Comment:
Same comment as for ELM JSON value set references. I found myself writing a very similar function for the CCSM Dashboard, and it could be nice to have this function handy somewhere.